core/qt-ble.cpp

```diff
@@ -490,7 +492,7 @@ dc_status_t BLEObject::setupHwTerminalIo(const QList<QLowEnergyCharacteristic> &
 	 * 0x0000FEFB is a clear indication that the OSTC is equipped with this BT/BLE hardware.
 	 */
 
-	if (allC.length() != 4) {
+	if (allC.length() != 4 && allC.length() != 2) {
 		qDebug() << "This should not happen. HW/OSTC BT/BLE device without 4 Characteristics";
```

Comment:
I guess we should update the debug message here as well... I can fix that when I merge it